.ci_helpers/docker/setup-services.py

```diff
@@ -0,0 +1,67 @@
+"""setup-services.py
```

Comment:
@emiliom You'd enjoy this script! Hopefully that will make it easier to bring up the necessary services. Try it out!

```bash
python .ci_helpers/docker/setup-services.py --help
```

I'm wondering if this should be an echopype CLI, but this is mostly for testing so idk. What do you think?

Reply:
> @emiliom You'd enjoy this script! Hopefully that will make it easier to bring up the necessary services. Try it out!

Nice indeed! I'll try it out, but not today.

Reply:
> I'm wondering if this should be an echopype CLI, but this is mostly for testing so idk. What do you think?

That's an interesting prospect. But my vote would be to banish any discussion of an echopype CLI until 0.5.0 is released! Otherwise, I fear it could be a distraction :disappointed:

Reply:
Hah, there _was_ an echopype CLI, but I removed it at some point. For the workflow stuff we will need it, and I don't think it's hard to put it in place, but like what @emiliom said, let's push that to after v5.0. :)